packages/chat/tests/Unit/Response/AIChatResponseStreamTest.php

```
@@ -77,4 +80,146 @@ public function testGetRequest(): void
 
         $this->assertSame($request->reveal(), $response->getRequest());
     }
+
+    public function testRegisterUsageCallback(): void
+    {
+        $request = $this->prophesize(AIChatStreamedRequest::class);
+        $receivedUpdates = [];
+
+        $callback = new class($receivedUpdates) implements UsageCallbackInterface {
+            public function __construct(private array &$updates)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$updates'. (undefined)

(UnusedFormalParameter)